thrifty/cost_variance.sp

```diff
@@ -4,6 +4,12 @@ variable "cost_explorer_service_cost_max_cost_units" {
   default     = 10
 }
 
+variable "cost_explorer_forecast_cost_max_cost_units" {
```

Comment:
Did you add this to `steampipe.spvars.example` and other docs/files that mention variables?